src/Controller/Api/MajorVersion/ReleasesController.php

```diff
@@ -112,28 +111,26 @@ public function getReleasesByMajorVersion(string $version, Request $request): Js
      * @SWG\Tag(name="major")
      * @SWG\Tag(name="release")
      *
-     * @param string|null $version Specific TYPO3 Version to fetch
-     * @return \Symfony\Component\HttpFoundation\JsonResponse
+     * @param string $version Specific TYPO3 Version to fetch
      */
     public function getLatestReleaseByMajorVersion(string $version, Request $request): JsonResponse
     {
         $this->checkMajorVersionFormat($version);
         $releaseRepo = $this->getDoctrine()->getRepository(MajorVersion::class);
-        /** @var MajorVersion $major */
         $major = $releaseRepo->findOneBy(['version' => $version]);
-        if (null === $major) {
+        if (!$major instanceof MajorVersion) {
```

Comment:
I'm curious as to Rector's reasoning for this type of change.  It's technically changing the meaning, no?  Or is it guaranteed that findOneBy() is returning `?MajorVersion`?

Reply:
Yes, the change is fine and I really like it. For this case there will always be `?MajorVersion` returned but for sure there are other examples where such a change really helps to ensure the correct class.